Program.cs

```diff
@@ -1,87 +1,26 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Linq.Expressions;
 using System.Runtime.InteropServices.ComTypes;
 using System.Text;
 using System.Threading.Tasks;
+using System.Web;
 
 namespace cookBook
 {
     internal class Program
```

Comment:
internal powinno się rzadko używać, jeśli coś musi być internal to trochę sugeruje że nie wiadomo co z tym zrobić, tutaj to nie ma aż takiego znaczenia, ale w przypadku klas które wydzieliłeś następnym krokiem jakiego będziemy się uczyć będzie stworzenie interfejsów. Jeśli mamy interfejs to klasy które go implementują będą publiczne, i to jest ok.